includes/components/class-admin.php

```diff
@@ -1681,6 +1681,49 @@ public function render_notices() {
 		// Add default notices.
 		$notices = [];
 
+        // Get unlicensed products.
+        $unlicensed_products = $this->get_unlicensed_products();
```

Comment:
Please try to simplify the code, e.g. we don't need 2 separate arrays for themes and extensions (if needed and the array item is a sub-array we can add a "type" parameter later), generally we need 3 steps:

1. iterate over the license keys and get an array of purchased products
2. iterate over the premium products and get an array of installed ones
3. compare arrays and if there are installed products not in the first array, add them to a list of products to show in a warning

We can do this without a helper for now, there will be a 1 month period for customers to resolve this